signer/src/testing/transaction_coordinator.rs

```diff
@@ -255,7 +256,10 @@ where
             .expect("Empty pending requests");
         let withdrawals = pending_requests.withdrawals;
 
-        // Get pending withdrawals from storage
+        // Get the chain tips from storage.
+        let (bitcoin_chain_tip_ref, stacks_chain_tip) = storage.get_chain_tips().await;
```

Comment:
This is already defined a few lines above.

Reply:
Indeed, merge gone wrong 😅 Removed in [8001b27](https://github.com/stacks-network/sbtc/pull/1399/commits/8001b27cce212a2586e986101622c163f3988a71) and also fixed a little with the new buffer value.